README.md

```diff
@@ -2,47 +2,64 @@
-| `language`                        | Optionally force associating a language with your coverage report                      | `''`                                  |
-| `force-coverage-parser`           | Optionally force using a specific coverage report parser                               | `''`                                  |
+    -   **If you're setting up one repository**, [obtain a project API token](https://docs.codacy.com/codacy-api/api-tokens/#project-api-tokens) and store it as an [encrypted secret for your **repository**](https://docs.github.com/actions/reference/encrypted-secrets#creating-encrypted-secrets-for-a-repository) with the name `CODACY_PROJECT_TOKEN`.
+    -   **If you're setting up multiple repositories**, [obtain an account API token](https://docs.codacy.com/codacy-api/api-tokens/#account-api-tokens) and store it as an [encrypted secret for your **organization**](https://docs.github.com/en/actions/reference/encrypted-secrets#creating-encrypted-secrets-for-an-organization) with the name `CODACY_API_TOKEN`.
+
+    > ⚠️ **Never write API tokens to your configuration files** and keep your API tokens well protected, as they grant owner permissions to your projects on Codacy.
+
+2.  [Generate a supported code coverage report](https://docs.codacy.com/coverage-reporter/#adding-coverage-to-your-repository) before each push to your repository.
+
+3.  Add the following to a file `.github/workflows/codacy-coverage-reporter.yaml` in your repository. Update the sample code with the name of the environment storing the access token and the paths to the report files generated previously:
+
+    ```yaml
+    name: Codacy Coverage Reporter
+
+    on: ["push"]
+
+    jobs:
+      codacy-coverage-reporter:
+        runs-on: ubuntu-latest
+        name: codacy-coverage-reporter
+        environment: production # Environment storing the access token
+        steps:
+          - uses: actions/checkout@v2
+          - name: Run codacy-coverage-reporter
+            uses: codacy/codacy-coverage-reporter-action@v1
+            with:
+              project-token: ${{ secrets.CODACY_PROJECT_TOKEN }}
+              # or
+              # api-token: ${{ secrets.CODACY_API_TOKEN }}
+              coverage-reports: path/to/coverage.xml
+              # or a comma-separated list for multiple reports
+              # coverage-reports: package1/coverage.xml, package2/coverage.xml
+              # language: '' # Optionally force associating a language with your coverage report
+              # force-coverage-parser: '' # Optionally force using a specific coverage report parser
+    ```
+
+4.  Optionally, to have information about the coverage results directly on your pull requests, [enable the GitHub integration](https://docs.codacy.com/repositories-configure/integrations/github-integration/) on Codacy.
```

Comment:
Currently, the only information about coverage that Codacy posts on pull requests are the **coverage status checks** that are mentioned in the last step (the users must configure a coverage quality gate and enable the status checks).

So, this step is actually redundant. :slightly_smiling_face: 

```suggestion
```